main.py

```diff
@@ -1,10 +1,42 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
```

Comment:
The context manager is currently being used for both reading the file and processing the data. According to the checklist, the context manager should only be used for reading data from the file. Move the processing logic outside of the context manager block.